src/web/app/pages-instructor/instructor-course-edit-page/instructor-course-edit-page.component.ts

```diff
@@ -380,6 +380,8 @@ export class InstructorCourseEditPageComponent implements OnInit {
     const instructorIsDisplayed: string = 'instructorisdisplayed';
     if (editedInstructor.isDisplayedToStudents) {
       paramsMap[instructorIsDisplayed] = 'true';
+    } else {
+      paramsMap[instructorIsDisplayed] = 'false';
```

Comment:
Why not:
```ts
// line 371
const paramsMap: { [key: string]: string } = {
  // ...
  instructorisdisplayed: editedInstructor.isDisplayedToStudents.toString(),
};
```

Reply:
Yeah, just saw that after I pushed the commit. 👍